docs/migrating/converting.asciidoc

```diff
@@ -52,16 +56,154 @@ Here's the recommended approach for converting an existing implementation to {ec
   smoothly in the future.
 
 [float]
-[[ecs-conv-spreasheet]]
-==== Using a spreadsheet to plan your migration
+[[ecs-conv-spreadsheet]]
+===== Use a spreadsheet to plan your migration
 
 Using a spreadsheet to plan the migration from pre-existing source fields to ECS
-is a common practice. It's a good way to address each of your fields methodically among colleagues.
+is a common practice. It's a good way to address each of your fields
+methodically among colleagues.
 
-If the data source is either a structured log, or if you already have a pipeline
-producing events with these non-ECS field names, the tool
-https://github.com/elastic/ecs-mapper[ECS Mapper] may help you get started in performing all of these field renames.
+To help you plan your migration, Elastic offers a
+https://ela.st/sample-pipeline-mapping[spreadsheet template]. You can use a CSV
+version of this spreadsheet to
+<<ecs-map-custom-data-to-ecs-es-pipeline,automatically create an {es} ingest
+pipeline>>. This is the easiest and most consistent way to map your custom data
+to ECS, regardless of your ingest method.
 
-After exporting your mapping spreadsheet to CSV, ECS Mapper will convert your field mapping
-to equivalent pipelines for Beats, Elasticsearch, and Logstash. Learn more at
-https://github.com/elastic/ecs-mapper[ECS Mapper].
+If you prefer to use {beats} or {ls} pipelines, you can also use the
```

Comment:
@kgeller @jamiehynds - I'm in favor of archiving `ecs-mapper` and indicating it's no longer maintained in the project's README. Unfortunately, the ECS team doesn't have the spare capacity to maintain or improve it right now.

If we archive the project, we can remove this reference from the docs.

Reply:
I pushed https://github.com/elastic/ecs/pull/1772/commits/bfaff544b18e7499572699a7b549bbba167b1960 to remove the reference. That lets us decouple these docs from archiving the `ecs-mapper` repo.